### ISSUE TITLE
docs: parameter and return types

### ISSUE THREAD

**Issue by jwmueller** [cleanlab/cleanlab#453]
Fix types in docstrings under:

Parameters
------------

Returns
--------

to utilize our new auto-typing where possible:  https://github.com/cleanlab/cleanlab/pull/412 

Sphinx can now automatically insert the type for us in the docstring (based on type annotation in the function signature), so you should not specify the type of Parameter or Return-variable *unless* it is a custom/nonstandard type that looks ugly when automatically inserted by Sphinx.  The best strategy would be to remove all the types (only in docstrings, not function signatures!) in a file and rebuild the docs to see how it looks. Then add back the subset of types that were not properly auto-inserted by Sphinx or simply don't look good.  

This has already been done for files token_classification module, which you can reference.
Remaining files to do/verify this for:

- [ ] classification.py
- [ ] count.py
- [ ] dataset.py
- [ ] filter.py
- [ ] multiannotator.py
- [ ] outlier.py
- [ ] rank.py
- [ ] experimental/keras.py
- [ ] experimental/fasttext.py
- [ ] experimental/coteaching.py
- [ ] experimental/mnist_pytorch.py
- [ ] experimental/cifar_cnn.py
- [ ] internal/label_quality_utils.py
- [ ] internal/latent_algebra.py
- [ ] internal/token_classification_utils.py
- [ ] internal/util.py
- [ ] internal/validation.py
- [ ] benchmarking/noise_generation.py

**Comment by ethanotran** [comment]
Can I work on this? Looking for my first open source contribution.

**Comment by jwmueller** [comment]
@ethanotran Yes definitely, thanks for your help!!  I'd just start with a single file. And make sure build the docs before/after your changes following our guide, so when you make the PR you can include a before/after screenshot as comments for easier review.
